simple-wp-helpdesk/src/Rest/TicketController.php

```
@@ -225,21 +260,31 @@ public function update_item( $request ): WP_REST_Response {
+			swh_update_ticket_fields(
+				$id,
+				$_field_patch,
+				array(
+					'source'   => 'rest_internal',
+					'actor_id' => get_current_user_id(),
+				)
+			);
+			// swh_set_ticket_status() bumps the version internally; explicit
+			// bump covers the priority/assignee branches that don't bump from
+			// the helper itself.
 			TicketVersion::bump( $id );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\SWH\Concurrency\TicketVersion' in method 'update_item'. (undefined)

(StaticAccess)